### PR TITLE
Update Google Pay direct integration

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/payment_methods/post-order-googlepay.md [hunk 1]
```diff
@@ -198,7 +200,12 @@ See [payment_options (object)](/api/#payment-options-object).
 ----------------
 `payment_url` | string 
 
-The URL of the page where the customer is redirected from your checkout to complete payment, which may be hosted by [MultiSafepay](/payment-pages/), the [issuer](/glossaries/multisafepay-glossary/#issuer), or the payment method.
+The URL of the page that you need to redirect your customer to.
```
Comment (FrancescaNCook) [inline content/api/payment_methods/post-order-googlepay.md:203]:
```suggestion
The URL of the page to redirect the customer to to complete payment. 
```

File: content/api/payment_methods/post-order-googlepay.md [hunk 2]
```diff
@@ -198,7 +200,12 @@ See [payment_options (object)](/api/#payment-options-object).
 ----------------
 `payment_url` | string 
 
-The URL of the page where the customer is redirected from your checkout to complete payment, which may be hosted by [MultiSafepay](/payment-pages/), the [issuer](/glossaries/multisafepay-glossary/#issuer), or the payment method.
+The URL of the page that you need to redirect your customer to. 
+
+Depending on [how to customer's card is stored](/payment-methods/google-pay/direct/#step-6-redirect-the-customer) in their Google Pay account, this URL references:
```
Comment (FrancescaNCook) [inline content/api/payment_methods/post-order-googlepay.md:205]:
```suggestion
Depending on [how the customer's card is stored](/payment-methods/google-pay/direct/#step-6-redirect-the-customer) in their Google Pay account, this URL references:
```

File: content/api/payment_methods/post-order-googlepay.md [hunk 3]
```diff
@@ -198,7 +200,12 @@ See [payment_options (object)](/api/#payment-options-object).
...
+
+- A 3D Secure authentication page. **or**
```
Comment (FrancescaNCook) [inline content/api/payment_methods/post-order-googlepay.md:207]:
```suggestion
- A 3D Secure authentication page, **or**
```

File: content/api/payment_methods/post-order-googlepay.md [hunk 4]
```diff
@@ -198,7 +200,12 @@ See [payment_options (object)](/api/#payment-options-object).
...
+- Your success page (`payment_options.redirect_url`).
```
Comment (FrancescaNCook) [inline content/api/payment_methods/post-order-googlepay.md:208]:
```suggestion
- Your success page (`payment_options.redirect_url`)
```

File: content/payments/methods/wallet/googlepay/direct-integration/_index.md [hunk 5]
```diff
@@ -290,6 +290,25 @@ From your server, create a [Google Pay direct order](/api/#google-pay---direct).
 
 For the `gateway_info.payment_token`, use `PaymentData.PaymentMethodData.PaymentMethodTokenizationData.token`.
 
+## Step 6: Redirect the customer
+
+In response to the API request you made in the previous step, you receive a `payment_url` (see API Reference – [Google Pay direct order](/api/#google-pay---direct)).
```
Comment (FrancescaNCook) [inline content/payments/methods/wallet/googlepay/direct-integration/_index.md:295]: Is there an extra bracket here?

File: content/payments/methods/wallet/googlepay/direct-integration/_index.md [hunk 6]
```diff
@@ -290,6 +290,25 @@ From your server, create a [Google Pay direct order](/api/#google-pay---direct).
...
+
+```
+document.location = payment_url
+```
+
+Depending on how the customer's card is stored in their Google Pay account, the URL references your success page, or a 3D Secure authentication page.
+
+If the customer's credit card was stored as:
+
+- **Token** (`CRYPTOGRAM_3DS`): the `payment_url` redirects to your succes page
```
Comment (FrancescaNCook) [inline content/payments/methods/wallet/googlepay/direct-integration/_index.md:307]:
```suggestion
- **Token** (`CRYPTOGRAM_3DS`), the `payment_url` redirects to your success page.
```

File: content/payments/methods/wallet/googlepay/direct-integration/_index.md [hunk 7]
```diff
@@ -290,6 +290,25 @@ From your server, create a [Google Pay direct order](/api/#google-pay---direct).
...
+- **Card on file** (`PAN_ONLY`): the `payment_url` may redirect to a 3D Secure authentication page
```
Comment (FrancescaNCook) [inline content/payments/methods/wallet/googlepay/direct-integration/_index.md:308]:
```suggestion
- **Card on file** (`PAN_ONLY`), the `payment_url` may redirect to a 3D Secure authentication page.
```

File: content/payments/methods/wallet/googlepay/direct-integration/_index.md [hunk 8]
```diff
@@ -290,6 +290,25 @@ From your server, create a [Google Pay direct order](/api/#google-pay---direct).
...
+
+If 3D Secure authentication is required, the customer is automatically redirected to your succes page after authentication.
```
Comment (FrancescaNCook) [inline content/payments/methods/wallet/googlepay/direct-integration/_index.md:310]:
```suggestion
If 3D Secure authentication is required, the customer is automatically redirected to your success page after authentication.
```

File: content/payments/methods/wallet/googlepay/direct-integration/_index.md (hunk 5 above)
Comment (Kris-MultiSafepay) [reply content/payments/methods/wallet/googlepay/direct-integration/_index.md:295]: One is markdown syntax and the other is linguistic syntax

**Pull request merged**